TurkishFilter/sections/general_extensions.txt

```diff
@@ -215,7 +215,7 @@ atv.com.tr#%#//scriptlet("set-constant", "HBiddings.vastUrl", "")
 kisalt.digital#%#!function(){if(-1<window.location.href.indexOf("/?u=aHR0c")){var a=location.href.split("/?u=");if(a[1])try{window.location=atob(a[1])}catch(b){}}}();
 ||kisalt.digital^$replace=/window\.location\.replace\(Link\);//,~script,~xmlhttprequest,~stylesheet
 ! https://github.com/AdguardTeam/AdguardFilters/issues/96719
-dizipal12.site,dizipal13.site,dizipal14.site,dizipal15.site,dizipal16.site,dizipal17.site,dizipal18.site,dizipal19.site,dizipal20.site,dizipal21.site,dizipal22.site,dizipal23.site,dizipal24.site,dizipal25.site,dizipal26.site,dizipal27.site,dizipal28.site,dizipal29.site,dizipal30.site,dizilla.*#%#//scriptlet('set-constant', 'wpsaData', 'undefined')
+dizipal12.site,dizipal13.site,dizipal14.site,dizipal15.site,dizipal16.site,dizipal17.site,dizipal18.site,dizipal19.site,dizipal20.site,dizipal21.site,dizipal22.site,dizipal23.site,dizipal24.site,dizipal25.site,dizipal26.site,dizipal27.site,dizipal28.site,dizipal29.site,dizipal30.site#%#//scriptlet('set-constant', 'wpsaData', 'undefined')
```

Comment:
not sure about it. The rule will quickly become obsolete, and in general I don't think this site is visited by users with blockers since no one is complaining.

Reply:
Or add a comment like 
`! dizipal*.cloud - frequently changed domains`
+ link where we can find an actual domain